packages/core/src/subscriber/chainSubscribers.ts

```diff
@@ -204,8 +208,9 @@ class ChainSubscribers {
 
         // send push notification
         if (user)
-            sendNotification([user], NotificationType.TRANSACTION_RECEIVED, true, true, {
-                path: NotificationParamsPath.TRANSACTION_RECEIVED
+            sendNotification([user], NotificationType.TRANSACTION_RECEIVED, true, true, undefined, undefined, {
+                amount,
+                asset
```

Comment:
It's just missing the transaction hash on the path, so we can show the bottom sheet straight away on the wallet. I think the parameter is correct https://docs.ethers.org/v5/api/providers/types/#providers-Log

```suggestion
                asset,
                path: log.transactionHash
```